site/docs/get-started/vagrant.md

```diff
@@ -19,10 +19,10 @@ Additional details on how to configure the system properly can be found in the l
 
 ### Step 2. Download and deploy the appliance
 
-1. Pull the ManageIQ Vagrant box from [Atlas](https://atlas.hashicorp.com/vagrant) and run it
+1. Pull the ManageIQ Vagrant box from [Atlas](https://atlas.hashicorp.com/ManageIQ) and run it
```

Comment:
I copied and pasted it from Atlas directly.
Using ManageIQ there gives the full list of boxes under ManageIQ.
But it works with lower cases, I will update it tomorrow